ui/src/router/components/form/components/ensembler_config/typeOptions.js

```diff
@@ -14,13 +14,26 @@ const typeOptions = [
   {
     value: "standard",
     inputDisplay: "Standard",
+    expenginetype: "standard",
     description: (
       <Fragment>
         Turing will select the response from one of the routes, based on the
         configured mapping between routes and experiment treatments
       </Fragment>
     ),
   },
+  {
+    value: "standard",
+    inputDisplay: "Standard",
+    expenginetype: "custom",
+    description: (
+      <Fragment>
+        Turing will select the route response corresponding to the route name specified
+        in the treatment configuration. The route name path will be used to locate the
+        name of the route within the treatment configuration.
+      </Fragment>
```

Comment:
Added a separate standard ensembler description for custom experiment engines.

Reply:
Do we _have to_ introduce this? The existing description seems generic enough to conceptually cover both cases, no?

> Turing will select the response from one of the routes, based on the configured mapping between routes and experiment treatments

Reply:
It may also help simplify some of the implementation (like https://github.com/caraml-dev/turing/pull/235/files#r976051034)

Reply:
Well, I was originally thinking that "configured mapping between routes and experiment treatments" doesn't accurately describe whatever we're going to display in the standard ensembler configuration panel, i.e. the "assigned routes" and "assigned experiments" (there's no mention of "treatment" anywhere). 

But technically, since we are expecting the experiment engines to implement the standard ensembler component to be consumed, I guess it makes sense that we should probably keep this description as generic as possible, and then expect more tooltips/information within the consumed component to explain what's going on.

Reply:
Yes, I think we can keep this description generic. Any additional details specific to the std / custom engines can be embedded within those components (like the Flyout we have for XP's remote component for the std ensembler). That way we can do away with the special handling based on `expenginetype` introduced in this file.

Reply:
Okay done, I've removed the standard ensembler option for custom experiment engines so it's now the same as before :)